readme.md

```diff
@@ -14,31 +14,15 @@ A [clamping function](https://en.wikipedia.org/wiki/Clamping_(graphics)) constra
 
 The primary motivation for this function is to bring parity with the [CSS function][css-clamp] of the same name. It will be useful when using it in conjunction with the [CSS Painting API](https://developer.mozilla.org/en-US/docs/Web/API/CSS_Painting_API).
 
-## Examples
+That is, given `Math.clamp(VAL, MIN, MAX)`, it represents exactly the same value as `Math.max(MIN, Math.min(VAL, MAX))`.
```

Comment:
```suggestion
That is, `Math.clamp(VAL, MIN, MAX)` is exactly equal to `Math.max(MIN, Math.min(VAL, MAX))`.
```

Reply:
This is copied from the CSS specification, which I think makes it more terminological.